packages/mux-player/src/html.ts

```diff
@@ -59,10 +59,11 @@ export function processBooleanNode(
 }
 
 export function processPart(part: TemplatePart, value: unknown): void {
-  processBooleanAttribute(part, value) ||
+  processDirective(part, value) ||
     processBooleanNode(part, value) ||
     processSubTemplate(part, value) ||
     processDocumentFragment(part, value) ||
+    processBooleanAttribute(part, value) ||
```

Comment:
any reason this order changed?

Reply:
I wanted processing the attributes together so it was easier to see because they're used now below too
https://github.com/muxinc/elements/pull/96/files#diff-41da7eb53a2250a1a0e038f4415e05489b9435fb1eba29153d6d2732d8973a52R181-R182